site/pom.xml

```diff
@@ -0,0 +1,108 @@
+         xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+    <modelVersion>4.0.0</modelVersion>
+
+    <parent>
+        <groupId>org.eclipse.hono</groupId>
+        <artifactId>hono-bom</artifactId>
+        <version>${revision}</version>
+        <relativePath>../bom</relativePath>
+    </parent>
+
+    <artifactId>hono-site</artifactId>
+    <packaging>pom</packaging>
+
+    <name>Hono Site</name>
+    <description>The content of the Hono website.</description>
+
+    <build>
+        <outputDirectory>${project.build.directory}/site</outputDirectory>
+        <resources>
+            <resource>
+                <directory>${project.basedir}</directory>
+                <targetPath>${project.build.directory}/site</targetPath>
+            </resource>
+        </resources>
+        <plugins>
+            <plugin>
+                <groupId>org.apache.maven.plugins</groupId>
+                <artifactId>maven-resources-plugin</artifactId>
+                <version>2.6</version>
+                <executions>
+                    <execution>
+                        <goals>
+                            <goal>copy-resources</goal>
+                        </goals>
+                        <phase>process-resources</phase>
+                        <configuration>
+                            <outputDirectory>${project.build.directory}/site</outputDirectory>
+                            <resources>
+                                <resource>
+                                    <targetPath>${project.build.directory}/site</targetPath>
+                                    <directory>${project.basedir}</directory>
+                                    <excludes>
+                                        <exclude>*.iml</exclude>
+                                        <exclude>*.md</exclude>
+                                        <exclude>*.xml</exclude>
+                                    </excludes>
+                                </resource>
+                            </resources>
+                        </configuration>
+                    </execution>
+                </executions>
+            </plugin>
+            <plugin>
+                <groupId>org.codehaus.mojo</groupId>
+                <artifactId>exec-maven-plugin</artifactId>
+                <version>1.6.0</version>
+                <executions>
+                    <execution>
+                        <goals>
+                            <goal>exec</goal>
+                        </goals>
+                        <phase>compile</phase>
+                    </execution>
+                </executions>
+                <configuration>
+                    <executable>/bin/bash</executable>
+                    <workingDirectory>${project.build.outputDirectory}</workingDirectory>
+                    <arguments>
+                        <argument>build-site.sh</argument>
```

Comment:
Yes, it does. I have added "set -e" to ensure that a failure in one script line will force the script to exit immediately.